### ISSUE TITLE
[Staging Review] Prepare accessibility artifact

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/va.gov-team#116029]
## Description
This ticket covers work to do an accessibility audit and corresponding a11y artifact, per requirements in the collaboration cycle staging review touchpoint.  

The audit should specifically cover the revised "start a new message flow" work going through collaboration cycle, and behind a feature flag in staging. 

## User story
- As an accessibility specialist, I want to review the secure message team's new designs that update the "start a new message" task flow so that I can flag any issues to the team, and so that I can complete a required artifact for them that is needed as part of the staging review process. 

## Notes
- [How to prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) 

## Possible tasks:
- [x] Download the Deque axe browser extension 
- [x] Reach out to other a11y specialists as needed for example accessibility artifacts (if helpful)
- [x] Complete the accessibility artifact

## Acceptance criteria
- [x] No violations indicated in axe scan results within the scope of the feature/product. (You can ignore any flagged issues that aren’t your responsibility.)
- [x] If axe scans detect issues that you are unable to resolve prior to your staging review, make sure these are documented in your accessibility testing artifact.
- [x] Accessibility artifact has been completed 
- [ ] Artifact is linked in the collaboration cycle ticket here under the "staging review" section.
- [x] Items identified by the axe scan that require fixes (if applicable) are ticketed SM backlog and PM of the team (Stacy Blackwood) is notified.

**Comment by sterkenburgsara** [comment]
@StacyB2023 we have an a11y specialist named Sarah Horton who will be joining soon, but she doesn't have VA credentials yet. I have notified her of this upcoming request and she can hopefully dive right in soon.
